### DISCUSSION TITLE
Cannot deploy ST Webhook

### DISCUSSION THREAD

**Discussion by seancpark** [iklein99/homebridge-smartthings#207] [Q&A]
I've been trying to setup the ST webhook and get to the point of deploy to test so it becomes available in the app and all I get is "Sorry. A temporary server error occurred. Please try again in a few minutes." for the last two days. I've generated another app thinking there might be a hidden glitch but it's the same response. 

Any thoughts? Thanks!

**Comment by iklein99** [comment]
What email address did you use when you registered on my site?Sent from my iPhoneOn Aug 22, 2023, at 6:44 PM, seancpark ***@***.***> wrote:﻿
I've been trying to setup the ST webhook and get to the point of deploy to test so it becomes available in the app and all I get is "Sorry. A temporary server error occurred. Please try again in a few minutes." for the last two days. I've generated another app thinking there might be a hidden glitch but it's the same response.
Any thoughts? Thanks!

**Reply by seancpark** [reply]
I haven't, was setting up smart app first.

**Comment by iklein99** [comment]
Ok. I’ll look into it  Sent from my iPhoneOn Aug 22, 2023, at 7:14 PM, seancpark ***@***.***> wrote:﻿
I haven't, was setting up smart app first.

**Comment by iklein99** [comment]
So I just did this and had no issue.  BE SURE to perform the VERIFY step outlined below and wait until it completes before trying to deploy to test.


<img width="720" alt="image" src="https://github.com/iklein99/homebridge-smartthings/assets/5103250/03bac768-2214-48a9-a88f-a6f02428a96d">

**Comment by seancpark** [comment]
Verified each time with no problem. Just got back, I'll log in and try to deploy it again.

**Reply by seancpark** [reply]
Tried to deploy again, same scenario. Investigating... Perhaps something is screwed up with my account.

**Reply by seancpark** [reply]
Something was wrong on the back end, nothing more elaborate was shared, but the team *just* fixed it and I was able to deploy and see webhook in the mobile app. Now time to setup the klein token and things should be up and running.
